versions/0_4/flake.nix

```diff
@@ -0,0 +1,31 @@
+{
+  inputs =
```

Comment:
Annoyingly this can't be left empty because the source cleaning needs it.. Which it shouldn't unless something tries to use the cleaned sources, so there's a logic error somewhere in the Nix code I suspect. I'm not going to try and track it, this is just a copy of 0_4_rc